### PR TITLE
Minor comment fixes & function extraction.

### PR DESCRIPTION
**Pull request by eileenmcnaughton**

### CODE REVIEW

File: CRM/Core/BAO/Phone.php [hunk 1]
```diff
@@ -171,7 +171,9 @@ public static function allPhones($id, $updateBlankLocInfo = FALSE, $type = NULL,
   }
 
   /**
-   * Get all the phone numbers for a specified location_block id, with the primary phone being first
+   * Get all the phone numbers for a specified location_block id, with the primary phone being first.
+   *
+   * This is called from CRM_Core_BAO_Block as a calculated function.
```
Comment (JoeMurray) [inline CRM/Core/BAO/Phone.php]: I don't think it is good form to say in comments where something is called from, as that can get dated.

**Comment by JoeMurray** [comment]
The code changes look fine, but I haven't tested to see if the refactoring of some code into two functions is working, nor checked to see if $tree was being accessed from outside the class inappropriately and thus making it protected causing a error to be raised.

**Comment by seamuslee001** [comment]
tested this and looks file to me

**Comment by seamuslee001** [comment]
@eileenmcnaughton have rebased it for you

**Comment by eileenmcnaughton** [comment]
added merge on based based on @seamuslee001 testing

**Pull request merged**